content/general/push/activate-subscribe.textile

```diff
@@ -44,31 +38,39 @@ Before you can activate your push device or receive push notifications, you must
 
 Installation, however, is platform-specific and as such, instructions for each platform and service are provided below:
 
-h3. Install Ably for Google Firebase Cloud Messaging on Android
+h3. On Android
 
-As with any Firebase-enabled app, you need to include one or more services within your app to handle interactions with Firebase. You need to have a service that extends "FirebaseMessagingService":https://firebase.google.com/docs/reference/android/com/google/firebase/messaging/FirebaseMessagingService (and overrides "@onMessageReceived()@":https://firebase.google.com/docs/reference/android/com/google/firebase/messaging/FirebaseMessagingService) in order to handle incoming push messages; the code that handles incoming push messages does not need to interact with Ably in any way.
+If you do not have a firebase project, create a new firebase project and add your Android app to the project. Then setup "firebase cloud messaging":https://firebase.google.com/docs/cloud-messaging/android/client in your Android application. Then "create and add a server key":https://knowledge.ably.com/where-can-i-find-my-google/firebase-cloud-messaging-api-key on your firebase project, and enter it in your Ably app's dashboard (App > Notifications tab > Push Notifications Setup > Setup Push Notifications). 
```

Comment:
```suggestion
If you do not have a Firebase project, create one and register your Android application with the project. "Set up Firebase Cloud Messaging":https://firebase.google.com/docs/cloud-messaging/android/client in your Android application.

Then, "generate a Cloud Messaging server key":https://knowledge.ably.com/where-can-i-find-my-google/firebase-cloud-messaging-api-key for your Firebase project, and configure your Ably app to use it by performing the following steps in the "Apps Dashboard":https://ably.com/dashboard 

* Select your Ably app
* Click the Notifications tab
* Configure Google/Firebase Cloud Messaging in "Push Notifications Setup"
```

Reply:
I went with "add your Android app to the project" because that is language used by Firebase in the UI and in documentation

https://support.google.com/firebase/answer/9326094?hl=en

![Screenshot 2021-08-02 at 15 48 35](https://user-images.githubusercontent.com/24711048/127880627-a2e36709-8e86-4da9-b66f-89f9e80a63c6.png)